### PR TITLE
kelly/ico-lykke

### PR DESCRIPTION
**Pull request by kellybinary**
Add Lykke Exchange section to ICO page

### CODE REVIEW

File: src/templates/landing_pages/ico.html.tt [hunk 1]
```diff
@@ -165,7 +165,19 @@
...
             </div>
-            <hr>
+            <div id="lykke-section" class="content-section center-text fill-bg-color">
+                <div class="container">
+                    <div class="gr-row">
+                        <div class="gr-10 gr-push-1">
+                            <h2>[% l('Trade your tokens on the Lykke Exchange') %]</h2>
+                            <img src="[% request.url_for('images/ico/icons/lykke_logo.svg') %]"/>
+                            <p>[% l('We\'ve reached an agreement to list the tokens from our forthcoming ICO on the [_1]. Lykke is a fintech company that\'s building a global marketplace on the blockchain.', '<a href="https://www.lykke.com/" target="_blank">Lykke Exchange</a>') %]</p>
+                            <a class="button" href="https://www.lykke.com/company/news/binary" target="_blank"><span>[% l('Read the official press release') %]</span></a>
```
Comment (negarn) [inline src/templates/landing_pages/ico.html.tt]: do we need the noreferrer noopener thing on these links?

Comment (kellybinary) [reply src/templates/landing_pages/ico.html.tt]: i think so? to be safe..

File: src/templates/landing_pages/ico.html.tt [hunk 2]
```diff
@@ -165,7 +165,21 @@
...
                 </div>
             </div>
-            <hr>
+            <div id="lykke-section" class="content-section center-text fill-bg-color">
+                <div class="container">
+                    <div class="gr-row">
+                        <div class="gr-10 gr-push-1">
+                            <h2>[% l('Trade your tokens on the Lykke Exchange') %]</h2>
+                            <img src="[% request.url_for('images/ico/icons/lykke_logo.svg') %]"/>
+                            <p>[% l('We\'ve reached an agreement to list the tokens from our forthcoming ICO on the [_1]. Lykke is a fintech company that\'s building a global marketplace on the blockchain.', '<a href="https://www.lykke.com/" target="_blank" rel="noreferrer noopener">Lykke Exchange</a>') %]</p>
```
Comment (negarn) [inline src/templates/landing_pages/ico.html.tt]: so we don't want to translate `Lykke Exchange`?

File: src/templates/landing_pages/ico.html.tt [hunk 3]
```diff
@@ -165,7 +165,21 @@
...
                 </div>
             </div>
-            <hr>
+            <div id="lykke-section" class="content-section center-text fill-bg-color">
+                <div class="container">
+                    <div class="gr-row">
+                        <div class="gr-10 gr-push-1">
+                            <h2>[% l('Trade your tokens on the Lykke Exchange') %]</h2>
+                            <img src="[% request.url_for('images/ico/icons/lykke_logo.svg') %]"/>
+                            <p>[% l('We\'ve reached an agreement to list the tokens from our forthcoming ICO on the <a href="[_1]" target="_blank" rel="noreferrer noopener">Lykke Exchange</a>. Lykke is a fintech company that\'s building a global marketplace on the blockchain.', 'https://www.lykke.com/') %]</p>
```
Comment (jy-binary) [inline src/templates/landing_pages/ico.html.tt]: Remove the sentence "Lykke is a fintech company that\'s building a global marketplace on the blockchain." - it's unnecessary.

File: src/templates/landing_pages/ico.html.tt [hunk 4]
```diff
@@ -165,7 +165,21 @@
...
                 </div>
             </div>
-            <hr>
+            <div id="lykke-section" class="content-section center-text fill-bg-color">
+                <div class="container">
+                    <div class="gr-row">
+                        <div class="gr-10 gr-push-1">
+                            <h2>[% l('Trade your tokens on the Lykke Exchange') %]</h2>
+                            <img src="[% request.url_for('images/ico/icons/lykke_logo.svg') %]"/>
+                            <p>[% l('We\'ve reached an agreement to list the tokens from our forthcoming ICO on the <a href="[_1]" target="_blank" rel="noreferrer noopener">Lykke Exchange</a>.', 'https://www.lykke.com/') %]</p>
```
Comment (jy-binary) [inline src/templates/landing_pages/ico.html.tt]: Actually please change the text to "We've reached an agreement to list the tokens on the Lykke Exchange."
(i.e. remove "from our forthcoming ICO")

Comment (kellybinary) [reply src/templates/landing_pages/ico.html.tt]: @jy-binary ok, text is updated

**Pull request merged**